rundeckapp/grails-app/services/rundeck/services/ScmService.groovy

```diff
@@ -1455,7 +1459,7 @@ class ScmService {
                 def newmeta = [name: job.getJobName(),groupPath: job.getGroupPath(),version: job.version, pluginMeta: jobStatus.commit.asMap()]
                 jobMetadataService.setJobPluginMeta(
                         job,
-                        STORAGE_NAME_IMPORT,
+                        STORAGE_NAME_EXPORT,
```

Comment:
if the metadata now uses the integration type to store separate metadata for each job, how does it work when you have both import and export configured, and you e.g. rename a job?  previously the metadata was used to track the sourceId when you import a job, and later when you export it. is the sourceId still passed from import to export in the same way?

Reply:
@gschueler 
with this code
https://github.com/rundeck/rundeck/pull/7030/files#diff-8af69461e09e7c99b0829830e738be41d7d6809725ea047db37711357c3381acR440-R443

using the git changes plus the sourceID (or ID) I am trying to detect that the job was renamed.

Reply:
here's the problem.  set up import using "archive" UUID behavior.  When imported, the uuid is removed, replaced with a new uuid, and the source uuid is stored in the job metadata (now under the "scm-import" name).  Then setup SCM export, using the export UUID behavior "original", this is meant to use the original source id in the export pattern, such as `${job.sourceId}.${config.format}`. However, the scm-export metadata does not contain the source Id anymore.  It used to work when the only metadata was stored with the "scm-import" metadata name.

Reply:
@gschueler I added more code to this case. The idea is to check (when the SCM export is enabled) if there is an SCM import metadata with a `sourceId`  attribute set